doc/Developer-How-To-Move-Classes-to-src.md

```diff
@@ -0,0 +1,105 @@
+How To Move Classes to `src`
+==============
+
+* [Home](help)
+  * [Developer Intro](help/Developers-Intro)
+
+Since April 2017, Friendica uses [Composer](help/Composer) to manage autoloading.
```

Comment:
Not really, as the `develop` branch -- where class moves are expected to happen -- is version-agnostic. This mention actually is extraneous, as this help page didn't exist before Composer was added.